WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CardPresentPaymentAlertsPresenter.swift

```diff
@@ -143,4 +144,14 @@ final class CardPresentPaymentAlertsPresenter: CardPresentPaymentAlertsPresentin
         dismissCommonAndPresent(animated: true)
         dismissSeveralFoundAndPresent(animated: true)
     }
+
+    func presentMerchantEducation(completion: @escaping () -> Void) {
+        let viewController = TapToPayEducationViewViewHostingController(onDismiss: completion)
+
+        if let modalController {
+            modalController.present(viewController, animated: true)
+        } else {
+            rootViewController?.present(viewController, animated: true)
+        }
```

Comment:
Could the app enter both cases? Just wanted to ensure both paths are tested.

Reply:
I could only detect a case where a modal is presented, so the second one is invalid. I will clarify the code.